plugin/trino-blackhole/src/main/java/io/trino/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -311,7 +313,7 @@ public Optional<ConnectorTableLayout> getNewTableLayout(ConnectorSession connect
     }
 
     @Override
-    public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorTableLayout> layout, RetryMode retryMode)
+    public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorTableLayout> layout, RetryMode retryMode, boolean replace)
```

Comment:
where is this parameter used?

Reply:
The original logic in this connector already supported table replacement because it depended on the engine logic. 
I added the code to use this argument now.